tmk_core/common/arm_atsam/printf.h

```diff
@@ -1,8 +1,8 @@
 #ifndef _PRINTF_H_
 #define _PRINTF_H_
 
-#define __xprintf dpf
 int dpf(const char *_Format, ...);
+#define __xprintf(fmt, ...) dpf(fmt, ##__VA_ARGS__)
```

Comment:
What's the reasoning behind this change?

Also, I suggest using `__VA_OPT__(,)` instead of `, ##` if the GCC version we use supports it, as it makes the intent clearer (comma omission in case of no arguments).

Reply:
In early debugging, I was having issues with getting all the print statements lined up so this was more a visual thing for myself. I'm reverting back to simply
```
#define __xprintf dpf
```

As for the ##, I'm following established coding.